### PR TITLE
New offliner procedure

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: doc-content/enterprise-only/openshift/offline-repo-proc.adoc [hunk 1]
```diff
@@ -65,18 +65,18 @@ endif::[]
 endif::offline_onprem[]
 . On the computer that has an outgoing connection to the public Internet, complete the following steps:
 .. Download the latest version of the http://release-engineering.github.io/offliner/[Offliner tool].
-.. Download the `{PRODUCT_FILE}-offliner.txt` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal.
-.. Enter the following command to use the Offliner tool to download the required artifacts:
+.. Download the `{PRODUCT_FILE}-offliner.zip` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/offline-repo-proc.adoc]: I know this isn't something you changed, but this isn't the way we should refer to links:
Download the latest version of the Offliner tool.

You don't need to change it for the release, but just so you know, I would do something like:
Download the latest version of the Offliner tool from the http://release-engineering.github.io/offliner/[Offliner @RedHat] website.

File: doc-content/enterprise-only/openshift/offline-repo-proc.adoc [hunk 2]
```diff
@@ -65,18 +65,18 @@ endif::[]
...
+.. Extract the contents of the `{PRODUCT_FILE}-offliner.zip` file into any directory.
+.. Change to the directory and enter the command:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/offline-repo-proc.adoc]: enter the following command:

File: doc-content/enterprise-only/openshift/offline-repo-proc.adoc [hunk 3]
```diff
@@ -65,18 +65,18 @@ endif::[]
...
 +
 [subs="attributes,verbatim,macros"]
 ----
-java -jar offliner-<version>.jar -r \https://maven.repository.redhat.com/ga/ -r \https://repo1.maven.org/maven2/ -d /home/user/temp {PRODUCT_FILE}-offliner.txt
+./offline-repo-builder.sh offliner.txt 
 ----
 +
-Replace `/home/user/temp` with an empty temporary directory and `<version>` with the version of the Offliner tool that you downloaded. The download can take a significant amount of time.
-
+This command creates a `repository` subdirectory and downloads the necessary artifacts into this subdirectory. Ignore the message that some downloads have failed.
```
Comment (winklerm) [inline doc-content/enterprise-only/openshift/offline-repo-proc.adoc]: Recent offliner manifest tests (for RHPAM/RHDM 7.7.0) report zero failed downloads. I am wondering if we could remove the suggestion to ignore failed downloads?

File: doc-content/enterprise-only/openshift/offline-repo-proc.adoc (hunk 1 above)
Comment (mramendi) [reply doc-content/enterprise-only/openshift/offline-repo-proc.adoc]: This part was actually removed from this section, but I will note it for the future, thanks.

**Pull request merged**